### ISSUE TITLE
Modified LootTable not updating

### ISSUE THREAD

**Issue by Wxrlds** [way2muchnoise/JustEnoughResources#279]
I use [Open Loader](https://www.curseforge.com/minecraft/mc-mods/open-loader) to change the LootTable of the Drowned Mob to drop Copper from Thermal instead of Gold Ingots but JER still shows that they only drop Gold.

Using:
Minecraft 1.16.5
Forge 36.0.24
OpenLoader-1.16.5-9.0.2
JustEnoughResources-1.16.4-0.12.0.107

**Comment by way2muchnoise** [comment]
Yeah, this has been a struggle point and related to #107 and #120

**Comment by lublak** [comment]
@way2muchnoise Possibly just ne manual configuration?
That my for who has a json in which you simply overwrite individual entities
